### DISCUSSION TITLE
Create New Asin

### DISCUSSION THREAD

**Discussion by Kernel1453** [amzn/selling-partner-api-models#3077] [General]
Hello All,
I'd like to create a new product/asin. Which api should i use and i need xml sample for learn which informations must send to amazon. Meanwhile, for create asin, i need to get Category and subcategories as per marketplace, how can i get categories from sp api?

Thank you.

**Comment by MeenaAmz** [comment]
Please refer to this [blog post](https://developer-docs.amazon.com/sp-api-blog/docs/manage-your-product-listings-lifecycle-with-selling-partner-api#selling-partner-apis-for-product-listings) on how to create\update listings. Individual APIs have a reference, model (swagger model) and use case guide with json samples in the [developer docs](https://developer-docs.amazon.com/sp-api/docs).
